sale_order_line_input/models/sale_order.py

```diff
@@ -51,7 +51,3 @@ def _onchange_order_partner_id(self):
             onchange_method(new_so)
         order_vals = new_so._convert_to_write(new_so._cache)
         self.order_id = SaleOrder.create(order_vals)
-
-    def action_sale_order_form(self):
```

Comment:
We have lost from 15.0 the ability to open the sales order form, so please restore it.